esrally/track/track.py

```diff
@@ -420,7 +420,7 @@ class OperationType(Enum):
     RawRequest = 5
     WaitForRecovery = 6
     CreateSnapshot = 7
```

Comment:
I think we could move `CreateSnapshot` again to administrative actions (i.e. anything > 1000) because Rally would not report request metrics by default for administrative operations (it's usually not interesting to know).

Reply:
Addressed in 32ee4abf6a15d4158992c9843958b51e9a766cb5